### ISSUE TITLE
Testcontainers hanging when using docker-compose on GitHub actions

### ISSUE THREAD

**Issue by mvillafuertem** [testcontainers/testcontainers-scala#129]
Hi all, 

I have problems using test containers with GitHub workflows,  any suggestion of how could I fix it ?

https://github.com/mvillafuertem/scala/runs/981935480?check_suite_focus=true

Thanks!

**Comment by LMnet** [comment]
Hi @mvillafuertem 

Could you try to minimize your example?

**Comment by mvillafuertem** [comment]
Hi @LMnet 

Thanks for your answer. Here is a simple example.

https://github.com/mvillafuertem/testcontainers-scala/actions

**Comment by LMnet** [comment]
@mvillafuertem 
```
16:07:36.424 WARN  [pool-13-thread-4    ] o.t.c.DockerComposeContainer         Unable to pre-fetch an image (localstack/localstack) depended upon by Docker Compose build - startup will continue but may fail. Exception message was: Could not pull image: write /var/lib/docker/tmp/GetImageBlob298969881: no space left on device
```
Looks like github actions are restricted in terms of space. Maybe there is some configuration needed?

**Comment by mvillafuertem** [comment]
@LMnet I don't think so, if you see this execution you can verify that the GitHub actions correctly execute the docker command.

https://github.com/mvillafuertem/testcontainers-scala/runs/997305748?check_suite_focus=true

I think it may be related to this #1473

**Comment by LMnet** [comment]
But what about `no space left on device` message?

Actually, I didn't work with github actions and can't help you with this. Also, I'm pretty sure that this issue is not related to testcontainers-scala. It's more about testcontainers-java, because testcontainers-scala is just a wrapper with scala API. I recommend you to try to raise an issue in testcontainers-java repository with the java example.

**Comment by mvillafuertem** [comment]
Because the Pepe class tries to download 12GB until it runs out of free space, and this is not correct. 

I have raised the issue in testcontainers-java repository. Thanks!!

**Comment by mvillafuertem** [comment]
Because the org.testcontainers.DockerClientFactory class tries to download 12GB until it runs out of free space, and this is not correct.

I have raised the issue in testcontainers-java repository. Thanks!!
